src/testWithSpringBoot_2_1/java/org/openrewrite/java/spring/NoRequestMappingAnnotationTest.java

```diff
@@ -281,7 +281,7 @@ public ResponseEntity<List<String>> getUsersPost(String userId) {
               
               @RestController
               public class UsersController {
-                  @PostMapping(value = "/user/{userId}/edit", produces = {MediaType.APPLICATION_JSON_VALUE})
+                  @PostMapping(value = "/user/{userId}/edit", produces = { MediaType.APPLICATION_JSON_VALUE })
```

Comment:
This one in particular, for me, was a "side-effect" of applying always auto-format and doing more changed than needed. The original parameter was in this format, but in the result, we are removing the spaces without reason. I thing it's better to preserve original format in this recipe.

Reply:
Agreed